docker-compose.zitadel.yml

```diff
@@ -0,0 +1,206 @@
+version: "3.7"
+
+services:
+  pg-compeng:
+    container_name: datalens-pg-compeng
+    image: postgres:13-alpine
+    environment:
+      POSTGRES_PASSWORD: "postgres"
+      POSTGRES_DB: postgres
+      POSTGRES_USER: postgres
+
+  control-api:
+    container_name: datalens-control-api
+    image: ghcr.io/datalens-tech/datalens-control-api:0.2091.0
+    environment:
+      BI_API_UWSGI_WORKERS_COUNT: 4
+      CONNECTOR_AVAILABILITY_VISIBLE: "clickhouse,postgres,chyt,ydb,mysql,greenplum"
+      RQE_FORCE_OFF: 1
+      DL_CRY_ACTUAL_KEY_ID: key_1
+      DL_CRY_KEY_VAL_ID_key_1: "h1ZpilcYLYRdWp7Nk8X1M1kBPiUi8rdjz9oBfHyUKIk="
+      RQE_SECRET_KEY: ""
+      US_HOST: "http://us:8083"
+      US_MASTER_TOKEN: "fake-us-master-token"
+      CONFIG_PATH: "/etc/backend-configs/api.yaml"
+
+      SERVICE_CLIENT_ID: "bi"
+      SERVICE_CLIENT_SECRET: "${BI_SERVICE_CLIENT_SECRET}"
+      DL_CLIENT_ID: "${DL_CLIENT_ID}"
+      DL_CLIENT_SECRET: "${DL_CLIENT_SECRET}"
+    ports:
+      - 8081:8080
+  
+    depends_on:
+      - us
+    volumes:
+      - ./backend-configs:/etc/backend-configs
+
+  data-api:
+    container_name: datalens-data-api
+    image: ghcr.io/datalens-tech/datalens-data-api:0.2091.0
+    environment:
+      GUNICORN_WORKERS_COUNT: 5
+      RQE_FORCE_OFF: 1
+      CACHES_ON: 0
+      MUTATIONS_CACHES_ON: 0
+      RQE_SECRET_KEY: ""
+      DL_CRY_ACTUAL_KEY_ID: key_1
+      DL_CRY_KEY_VAL_ID_key_1: "h1ZpilcYLYRdWp7Nk8X1M1kBPiUi8rdjz9oBfHyUKIk="
+      BI_COMPENG_PG_ON: 1
+      BI_COMPENG_PG_URL: "postgresql://postgres:postgres@pg-compeng:5432/postgres"
+      US_HOST: "http://us:8083"
+      US_MASTER_TOKEN: "fake-us-master-token"
+      CONFIG_PATH: "/etc/backend-configs/api.yaml"
+
+      SERVICE_CLIENT_ID: "bi"
+      SERVICE_CLIENT_SECRET: "${BI_SERVICE_CLIENT_SECRET}"
+      DL_CLIENT_ID: "${DL_CLIENT_ID}"
+      DL_CLIENT_SECRET: "${DL_CLIENT_SECRET}"
+    ports:
+      - 8082:8080
+  
+    depends_on:
+      - us
+      - pg-compeng
+    volumes:
+      - ./backend-configs:/etc/backend-configs
+
+  pg-demo-connection:
+    container_name: datalens-pg-demo-connection
+    image: postgres:13-alpine
+    environment:
+      POSTGRES_DB: demo
+      POSTGRES_USER: demo
+      POSTGRES_PASSWORD: demo
+    volumes:
+      - ./pg-demo-connection/init:/docker-entrypoint-initdb.d
+      - ./pg-demo-connection/data:/var/lib/postgresql/data
+
+  pg-us:
+    container_name: datalens-pg-us
+    image: postgres:13-alpine
+    environment:
+      POSTGRES_DB: us-db-ci_purgeable
+      POSTGRES_USER: us
+      POSTGRES_PASSWORD: us
+    volumes:
+      - ./metadata:/var/lib/postgresql/data
+ 
+    ports:
+      - "8086:5432"
+
+
+  us:
+    container_name: datalens-us
+    image: ghcr.io/datalens-tech/datalens-us:0.209.0
+    depends_on:
+      - pg-us
+    environment:
+      APP_INSTALLATION: "opensource"
+      APP_ENV: "prod"
+      MASTER_TOKEN: "fake-us-master-token"
```

Comment:
Add auto generation of US master token plz

Reply:
done